Library/Formula/slimerjs.rb

```diff
@@ -24,27 +22,34 @@ def install
       system "zip", "-r", "omni.ja", "chrome/", "components/", "modules/",
                     "defaults/", "chrome.manifest", "-x@package_exclude.lst"
     end
-    libexec.install ["application.ini", "omni.ja", "slimerjs", "slimerjs.py"]
+    libexec.install %w[application.ini omni.ja slimerjs slimerjs.py]
     bin.install_symlink libexec/"slimerjs"
   end
 
   def caveats; <<-EOS.undent
     You can set the environment variable SLIMERJSLAUNCHER to a installation of
-    Mozilla Firefox (or Mozilla XULRunner) to use this version with SlimerJS instead
-    of the one installed by Homebrew (this is required if build without xulrunner).
+    Mozilla Firefox (or Mozilla XULRunner) to use this version with SlimerJS
+    instead of the one installed by Homebrew (this is required if built without
+    xulrunner).
     For a standard Mozilla Firefox installation this would be:
 
       export SLIMERJSLAUNCHER=/Applications/Firefox.app/Contents/MacOS/firefox
 
-    Note: If you use SlimerJS with an unstable version of Mozilla Firefox/XULRunner
-    (>29.*) you may have to change the [Gecko]MaxVersion in:
+    Note: If you use SlimerJS with an unstable version of Mozilla
+    Firefox/XULRunner (>29.*) you may have to change the [Gecko]MaxVersion in:
       #{libexec}/application.ini
     EOS
   end
 
   test do
-    system "#{bin}/slimerjs", "-v"
-    curl "-O", "https://raw.githubusercontent.com/laurentj/slimerjs/ec1e53a/examples/phantomjs/loadspeed.js"
-    system "#{bin}/slimerjs", "loadspeed.js", "https://www.google.com"
+    firefox = "/Applications/Firefox.app/Contents/MacOS/firefox"
+    has_firefox = File.executable? firefox
```

Comment:
We can assume `firefox` isn't around as this should be optimised for running on the bot which won't have it.